Documentation/ceph-pool-crd.md

```diff
@@ -31,6 +31,26 @@ spec:
   deviceClass: hdd
 ```
 
+#### Hybrid Storage Pools
+Hybrid storage is a combination of two different storage tiers. For example, SSD and HDD.
+This helps to improve the read performance of cluster by placing, say, 1st copy of data on the higher performance tier (SSD or NVME) and remaining replicated copies on lower cost tier (HDDs).
+
+```yaml
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+  name: replicapool
+  namespace: rook-ceph
+spec:
+  failureDomain: host
+  replicated:
+    size: 3
+    hybridStorage:
+      primaryDeviceClass: ssd
+      secondaryDeviceClass: hdd
+```
+Note: Ensure that `primaryDeviceClass` and `secondaryDeviceClass` are valid and they have at least one OSD associated with them, else the crush rule re-compile will fail.
```

Comment:
I agree with the warning although I'll remove the `else the crush rule re-compile will fail.
Write`, it feels like an internal details.
```suggestion
Warning: Ensure that `primaryDeviceClass` and `secondaryDeviceClass` are valid and they have at least one OSD associated with them, else the crush rule re-compile will fail.
```